curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/regular-expressions.english.md

```diff
@@ -17,13 +17,20 @@ videoId: Yud_COr6pZo
 
 ```yml
 question:
-  text: 'Which regex matches a white space character?'
+  text: |
+    Which regex only matches a white space character?
+
   answers:
-    - '\S'
-    - '\s'
-    - '.'
-    - '\_'
-    - '\.'
+    - |
+        \S
+    - |
+        \s
+    - |
+        .
+    - |
+        \_
+    - |
+        \.
```

Comment:
```suggestion
        \\.
```
For some reason \\. escapes the dot (not sure why), but that means you need an extra slash.